docs/firefox-accounts.rst

```diff
@@ -54,31 +54,13 @@ Local Development
 Demo Server Testing
 -------------------
 
-#. Quit Firefox.
-#. Create a new profile for testing called ``FxA Test Demo`` by following the
-   `instructions here`_.
-#. In the new profile folder, create a file called ``user.js`` and paste in the
-   following content:
-
-    .. code-block:: javascript
-
-        user_pref("services.sync.log.appender.file.logOnSuccess", true);
-        user_pref("identity.fxaccounts.auth.uri", "https://api-accounts.stage.mozaws.net/v1");
-        user_pref("identity.fxaccounts.remote.force_auth.uri", "https://accounts.stage.mozaws.net/force_auth?service=sync&context=fx_desktop_v1");
-        user_pref("identity.fxaccounts.remote.signin.uri", "https://accounts.stage.mozaws.net/signin?service=sync&context=fx_desktop_v1");
-        user_pref("identity.fxaccounts.remote.signup.uri", "https://accounts.stage.mozaws.net/signup?service=sync&context=fx_desktop_v1");
-        user_pref("identity.fxaccounts.settings.uri", "https://accounts.stage.mozaws.net/settings");
-        user_pref("identity.fxaccounts.remote.webchannel.uri", "https://accounts.stage.mozaws.net/");
-        user_pref("services.sync.tokenServerURI", "https://token.stage.mozaws.net/1.0/sync/1.5");
-        user_pref("webchannel.allowObject.urlWhitelist", "https://accounts.firefox.com https://accounts.stage.mozaws.net https://content.cdn.mozilla.net https://input.mozilla.org https://support.mozilla.org https://install.mozilla.org https://stomlinson.dev.lcip.org/");
-
-        user_pref("general.warnOnAboutConfig", false);
-        user_pref("devtools.chrome.enabled", true);
-        user_pref("devtools.debugger.remote-enabled", true);
-
-#. Start Firefox using the new profile you created in step 2.
-#. Verify the FxA settings look correct by opening ``about:config`` and searching for
-   ``identity.fxaccounts``.
+#. Open ``about:profiles``.
```

Comment:
@jpetto I added some handy new steps for testing the FxA form on demo that [Ryan pointed me to in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1405822#c28). I updated the docs here, but would be good if you could try these steps out too. Seems much simpler? :)

A didn't update the instructions for local testing, because they seem, well... broken for a long time (should we even be keeping them around)?

Reply:
Also, fwiw I don't seem to get verification emails for Mailinator any longer, although other email addresses seem to work fine.

Reply:
I think Mailinator addresses are now flagged/blocked on the FxA side.

Reply:
Wow these new instructions are so much easier. Excellent.

I would say we can drop the local instructions for now. AFAIK, they *have* been broken for a very long time. It would be nice to get them fixed (as pushing to a demo for testing isn't the fastest), but I don't see a reason to keep broken instructions around. Maybe we file a bug for local testing?